cernopendata_client/cli.py

```diff
@@ -152,7 +152,8 @@ def get_file_locations(server, recid, doi, title, protocol, expand):
     type=click.STRING,
     help="Which CERN Open Data server to query? [default=http://opendata.cern.ch]",
 )
-def download_files(server, recid, doi, title, protocol, expand):
+@click.option("--file", nargs=2, type=str, help="")
+def download_files(server, recid, doi, title, protocol, expand, file):
```

Comment:
- [ ] Can you please add help message to document the option?  
- [ ] Also, add some example below in the docstring, following the current ones.
- [ ] Also, update the `docs/usage.rst` user guide mentioning this possibility.
- [ ] Coverage decreased as part of this PR, which is OK, but if you separate filtering functions outside the main entry point function, then you could write test cases for those.
- [ ] Note that we may need to introduce several options, e.g. if somebody ask for `--filename '*.root'` there could be O(3000) files, so it makes sense to also offer navigation by file index number.  So either we have one option and call it filter:

```
--filter name '*.root' --filter range 10-19
```

or we have several options:

```
--filter-filename '*.root' --filter-files 10-19
``` 
```